### PR TITLE
Send icons: icon size, app chooser

### LINKED ISSUE #2248: Sharing: app icons are not sized equally
### Actual behaviour
The app icons in the sharing menu are not equally sized, instead they are displayed in their original size.
2 screenshots:
screenshot_20180228-110941
screenshot_20180228-112346

### Expected behaviour
The app icons in the sharing menu should be equally sized.
 
### Steps to reproduce
1. Click 'Share' icon on any file

### Environment data
Android version: 8.1.0

Device model: Google Pixel 2

Stock or customized system: stock

Nextcloud app version: 3.0.1

Nextcloud server version: 13.0.0

### Logs
 (n/a)

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Fix #2248

- equal sized icon
- fix showing app chooser twice

@xXSTrikeXx I also did not found a way to set a size to icons on TextView, so I added a new ImageView and wrapped this in a LinearLayout.

As it is a bug fix it should be in 3.1.

Signed-off-by: tobiaskaminsky <tobias@kaminsky.me>

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
:+1: besides the build failure due to raised lint score (my guess would be because of the removed compound drawable)

**Comment by AndyScherzinger** [comment]
pinging @mario for review :)

File: src/main/res/values/strings.xml [hunk 1]
```diff
@@ -772,4 +772,5 @@
     <string name="untrusted_domain">Access through untrusted domain. Please see documentation for further info.</string>
     <string name="notification_channel_push_name">Push notifications</string>
     <string name="notification_channel_push_description">Show push notifications sent by server, e.g. when mentioned in comments, you receive a new remote share or an announcement was posted by an admin.</string>
+    <string name="sendbutton_description">Icon for send button</string>
```
Comment (mario) [inline src/main/res/values/strings.xml]: "Send button icon" please.

Comment (AndyScherzinger) [reply src/main/res/values/strings.xml]: DONE https://github.com/nextcloud/android/pull/2331/commits/7626bfd7d22553d02ef27ff2783f9664339e3107

**Comment by AndyScherzinger** [comment]
@mario fixed coments and rebased for approval :)

**Comment by mario** [comment]
Merge

**Pull request merged**